README.md

```diff
@@ -1,4 +1,4 @@
-![logo](https://github.com/puzzle/cryptopus/blob/master/app/webpacker/images/cryptopussy.svg)
+![logo](https://raw.githubusercontent.com/puzzle/cryptopus/57f8ad8de410e4a0ba16227620727787f22c7d1c/frontend/public/assets/images/cryptopussy.svg)
```

Comment:
While not in scope of this pull request, still highly appreciated as it annoyed me for the last few weeks :grin: